public/javascripts/main.js

```diff
@@ -39,3 +40,9 @@ function signoutListener() {
     });
   }
 }
+
+function runPrettify() {
+  [].forEach.call(document.querySelectorAll('.entry-content pre'), (pre) => {
+    hljs.highlightBlock(pre);
```

Comment:
I think we can just do

```
hljs.initHighlightingOnLoad();
```

since hightlight.js by default highlights `<pre><code>...</code></pre>`, which is correctly generated from markdown.